### ISSUE TITLE
CookieContenair

### ISSUE THREAD

**Issue by messi89** [Vinelab/http#24]
Hi

is this package support hanling cookies from server ? (like CookieContenair)

**Comment by Mulkave** [comment]
Not really, this package is for simple HTTP(S) requests/response, but you can still retrieve and parse headers yourself. Do you have any suggestion for adding such feature? We're open for suggestions and PRs
